try {
  const a = 1 * '2';
  console.log(a);
} catch (e) {
  console.log(e);
}
